docs/user-guide/catalog.md

```diff
@@ -429,3 +429,18 @@ In **General Settings** of a IFC source type, it is possible to specify the serv
     * *Zoom to selected layer extent* <img src="../img/button/zoom-layer.jpg" class="ms-docbutton"/>: in order to zoom the map to the layer's extent
     * Access the [Layer Settings](layer-settings.md#layer-settings) <img src="../img/button/properties.jpg" class="ms-docbutton"/> to view/edit the [General Information](layer-settings.md#general-information) and the [Display](layer-settings.md#ifc-layer) options
     * *Remove* the layer <img src="../img/button/delete.jpg" class="ms-docbutton"/>
+
+### ArcGIS Catalog
+
+An [**ArcGIS**](https://enterprise.arcgis.com/en/server/latest/publish-services/windows/services-in-arcgis-enterprise.htm) service organize spatial data into functional GIS resources to run on a web server. MapStore allows to add ArcGIS service through its *Catalog* tool where a specific source type can be configured.
```

Comment:
I think we should specify that we are supporting ArcGIS Rest Service (https://developers.arcgis.com/rest/services-reference/enterprise/get-started-with-the-services-directory/). In particular we support MapServer and ImageServer types.

```suggestion
An [**ArcGIS Server Services Directory**](https://developers.arcgis.com/rest/services-reference/enterprise/get-started-with-the-services-directory/) is a RESTful representation of all the services running on an ArcGIS Server site. MapStore allows to add ArcGIS [Map Service](https://developers.arcgis.com/rest/services-reference/enterprise/map-service/) and [Image Service](https://developers.arcgis.com/rest/services-reference/enterprise/image-service/) types through its *Catalog* tool where a specific source type can be configured.
```